features/evaluation.feature

```diff
@@ -1,67 +0,0 @@
-Feature: Flag evaluation
```

Comment:
we should not remove this yet. there are other references for these definitions. for example - https://github.com/open-feature/java-sdk-contrib/blob/main/providers/flagd/pom.xml#L211-L212

Reply:
They are bound to old releases, not main. This is breaking, but it won't suddenly break any dependency repos, they are bound to release tags.

Reply:
🤔  hmm, then we should avoid merging renovate PRs of dependent repositories

Reply:
🤔  then we should avoid updating submodules on dependent repositories

Reply:
Yes those repositories should instead use the spec now, which has these gherken suites. I have a WIP PR for the js-sdk-contribs.